esphome/components/usb_cdc_acm/__init__.py

```diff
@@ -0,0 +1,76 @@
+import esphome.codegen as cg
+from esphome.components import esp32, uart
+from esphome.components.esp32 import add_idf_sdkconfig_option
+from esphome.components.esp32.const import (
+    VARIANT_ESP32P4,
+    VARIANT_ESP32S2,
+    VARIANT_ESP32S3,
+)
```

Comment:
https://github.com/esphome/esphome/pull/12243
```suggestion
from esphome.components.esp32 import (
    add_idf_sdkconfig_option,
    VARIANT_ESP32P4,
    VARIANT_ESP32S2,
    VARIANT_ESP32S3,
)
```

Reply:
🤦🏻‍♂️ Thank you 🙂